gtfs_realtime_translators/bindings/intersection_pb2.py

```diff
@@ -134,18 +175,18 @@
 DESCRIPTOR.extensions_by_name['intersection_stop_time_update'] = intersection_stop_time_update
 _sym_db.RegisterFileDescriptor(DESCRIPTOR)
 
-IntersectionTripUpdate = _reflection.GeneratedProtocolMessageType('IntersectionTripUpdate', (_message.Message,), {
-  'DESCRIPTOR' : _INTERSECTIONTRIPUPDATE,
-  '__module__' : 'intersection_pb2'
+IntersectionTripUpdate = _reflection.GeneratedProtocolMessageType('IntersectionTripUpdate', (_message.Message,), dict(
+  DESCRIPTOR = _INTERSECTIONTRIPUPDATE,
```

Comment:
Just to make sure: this was done by the proto -> py conversion automatically, not by hand, right?

Reply:
I didn't manually change these files, but I'll double-check that the IDE didn't modify them.

Reply:
Confirmed this was only modified from the conversion.